GoMeet/src/main/java/com/google/sps/servlets/VoteMeetingTimeServlet.java

```diff
@@ -0,0 +1,99 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.sps.data.ErrorMessages;
+import com.google.sps.data.MeetingTimeFields;
+import com.google.sps.data.ServletUtil;
+import java.io.IOException;
+import java.lang.IllegalArgumentException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet to handle the fetching and creating of MeetingTime entities */
+@WebServlet("/vote-meeting-time")
+public class VoteMeetingTimeServlet extends HttpServlet {
+
+  /** Increments the voteCount of the MeetingTime identified by the provided MeetingTimeId */
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // expect the query string to be of the following format: ?meetingTimeId=ID&voters=VOTER
+    String keyStr = request.getParameter(MeetingTimeFields.MEETING_TIME_ID);
+    String voter = request.getParameter(MeetingTimeFields.VOTERS);
+
+    if (keyStr == null || voter == null) {
+      ServletUtil.sendErrorResponse(response, HttpServletResponse.SC_BAD_REQUEST, ErrorMessages.BAD_REQUEST_ERROR);
+      return;
+    }
+
+    Key key;
+    try {
+      key = KeyFactory.stringToKey(keyStr);
+    } catch(IllegalArgumentException e) {
+      // the keyStr is an invalid key
```

Comment:
supernit: please try to have a consistent way of writing comments, e.g. capitalized first character or not. It'll make the code looks better

Reply:
Yep, capitalised all of the comments now 😊